mockzilla-management-ui/common/src/commonMain/kotlin/com/apadmi/mockzilla/desktop/engine/connection/NetworkingUtils.kt

```diff
@@ -16,7 +15,7 @@ fun Enumeration<NetworkInterface>.isLocalIpAddress(
     networkInterface.inetAddresses.toList().any { it.hostAddress == address }
 }
 
-suspend fun Enumeration<NetworkInterface>.findMdnsAddress(): InetAddress? = toList()
+suspend fun Enumeration<NetworkInterface>.findMdnsAddresses() = toList()
```

Comment:
I'd generally keep the return type, especially when it's not a type defined in our code